### ISSUE TITLE
pnpm command not found

### ISSUE THREAD

**Issue by NetiTrinR** [rstacruz/pnpm#137]
After installing I went to run pnpm install lodash. pnpm command wasn't found however.

Did I miss a step?

**Comment by iamstarkov** [comment]
did you install it globally?

**Comment by NetiTrinR** [comment]
I did. Checking npm's directory I can see it installed.

**Comment by rstacruz** [comment]
`npm install -g pnpm`? can you do `which pnpm`?

**Comment by rstacruz** [comment]
also are you using something like `n` or `nvm` or `asdf` or...?

**Comment by NetiTrinR** [comment]
Not sure what happened (Again I could see that it was installed). But I ran
npm install -g pnpm again and now it works. Probably something on my system
just messed up.

Thanks for all the support!

**Comment by iamstarkov** [comment]
@NetiTrinR can you close an issue?

**Comment by bengle** [comment]
> Not sure what happened (Again I could see that it was installed). But I ran npm install -g pnpm again and now it works. Probably something on my system just messed up.
> 
> Thanks for all the support!
> 
> On Tue, Mar 1, 2016 at 1:00 PM Rico Sta. Cruz [notifications@github.com](mailto:notifications@github.com) wrote:
> 
> > also are you using something like n or nvm or asdf or...?
> > —
> > Reply to this email directly or view it on GitHub
> > [#137 (comment)](https://github.com/pnpm/pnpm/issues/137#issuecomment-190879061).

Same problem,so which version of node are you using?
I find that version 16 is ok,but version 14 not work.

**Comment by unnimona** [comment]
same problem
this works
npm install -g pnpm

**Comment by plwdxiiu** [comment]
same problem.
It seems like it doesn't work when I was using node v14. 
check the version to 16 just fine.

**Comment by sursabipul** [comment]
> **First, use pnpm standalone script. It will worked.**
[https://pnpm.io/installation](url)

If that didn't work then copy pnpm.exe file from

>    C:\Users\hp\AppData\Local\pnpm

folder and paste it into

  >  C:\Program Files\nodejs
    
  folder.

**Comment by bipulmgr** [comment]
> **First, use pnpm standalone script. It will worked.**
[https://pnpm.io/installation](url)

If that didn't work then copy pnpm.exe file from

>    C:\Users\hp\AppData\Local\pnpm

folder and paste it into

  >  C:\Program Files\nodejs
    
  folder.

**Comment by black15** [comment]
Install it globally on your system using the:
```wget -qO /bin/pnpm "https://github.com/pnpm/pnpm/releases/latest/download/pnpm-linuxstatic-x64" && chmod +x /bin/pnpm```

**Comment by 1mehdifaraji** [comment]
I had the same problem which in my case I did change my system node version with ```nvm``` and reverting it back to the version that I already installed ```pnpm``` in the past , fixed the issue .

**Comment by huangsijun17** [comment]
My pnpm is also not working.

![image](https://user-images.githubusercontent.com/31504595/230269523-d38243b2-6b46-46d5-91ac-2bf61db872bf.png)

```
[root@VM-20-16-centos ~]# npm install -g pnpm
/www/server/nodejs/v16.20.0/bin/pnpm -> /www/server/nodejs/v16.20.0/lib/node_modules/pnpm/bin/pnpm.cjs
/www/server/nodejs/v16.20.0/bin/pnpx -> /www/server/nodejs/v16.20.0/lib/node_modules/pnpm/bin/pnpx.cjs
npm WARN notsup Unsupported engine for pnpm@8.1.1: wanted: {"node":">=16.14"} (current: {"node":"14.17.6","npm":"6.14.18"})
npm WARN notsup Not compatible with your version of node/npm: pnpm@8.1.1

+ pnpm@8.1.1
added 1 package in 8.279s
[root@VM-20-16-centos ~]# pnpm
-bash: pnpm: command not found
[root@VM-20-16-centos ~]# which pnpm
/usr/bin/which: no pnpm in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin)
[root@VM-20-16-centos ~]# node --version
v14.17.6
[root@VM-20-16-centos ~]# npm --version
6.14.18
```

**Comment by lightbluepoppy** [comment]
The same problem occurred.

**Comment by pootytang** [comment]
I recently had this same problem on linux mint 21.  I've been running pnpm 8.7.6 globally and got this today:
-------------------------------------------------------------------------------------------------
Update available! 8.7.6 → 8.9.2.                 │
   │   Changelog: https://github.com/pnpm/pnpm/releases/tag/v8.9.2   │
   │                Run "pnpm add -g pnpm" to update.                │
   │                                                                 │
   │     Follow @pnpmjs for updates: https://twitter.com/pnpmjs
-------------------------------------------------------------------------------------------------

So I ran the command it suggests like I have in the past but this time I see this:
-------------------------------------------------------------------------------------------------
pnpm add -g pnpm
Nothing to stop. No server is running for the store at /home/delane/.local/share/pnpm/store/v3
Packages: +1 -1
+-
 ERROR  Worker pnpm#1 exited with code 1
-------------------------------------------------------------------------------------------------

Now running the upgrade command again produces this:
-------------------------------------------------------------------------------------------------
pnpm add -g pnpm
zsh: command not found: pnpm
-------------------------------------------------------------------------------------------------


$ which pnpm
 pnpm not found

$ node --version
v18.14.0

Running npm install -g pnpm resolved it but weird that the ugprade seemed to remove it or possible change the permissions so that its not executable???

**Comment by PauloRWM** [comment]
> wget -qO /bin/pnpm "https://github.com/pnpm/pnpm/releases/latest/download/pnpm-linuxstatic-x64" && chmod +x /bin/pnpm

deu certo
